src/header/header.scss

```diff
@@ -13,8 +13,7 @@
   user-select: none;
```

Comment:
Could you undo formatting because it is hard to review?

Reply:
That's why I created #372

Reply:
Done. Also I suggest hiding whitespace changes in git diff